modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -14,7 +14,10 @@ As a network administrator, you can configure a network attachment definition of
 
 .Procedure
 
-. Create a network attachment definition manifest.
+. Create a network attachment definition manifest. Ensure that you create the network attachment definition in the same namespace as the virtual machine.
```

Comment:
I think we could make this a bit more concise. "manifest" tends to be optional. 

I would probably phrase this, "Create a network attachment definition in the same namespace as the virtual machine."

Reply:
Changed step to: "Create a network attachment definition in the same namespace as the virtual machine."